components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/controller/SpecificationsController.java

```diff
@@ -19,60 +18,96 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.UUID;
+
 @Controller
 public class SpecificationsController extends com.sap.cloud.cmp.ord.service.controller.Controller {
 
     private final String NOT_FOUND_MESSAGE = "Not Found";
     private final String INVALID_TENANT_ID_ERROR_MESSAGE = "Missing or invalid tenantID";
 
+    private static final String UNAUTHORIZED_MSG = "{\n" +
+            "  \"error\": {\n" +
+            "    \"code\": 401,\n" +
+            "    \"status\": \"Unauthorized\",\n" +
+            "    \"request\": \"8ed540a8-b4c6-49e3-a5ca-8dc29ba94318\",\n" +
```

Comment:
What's the semantic of this hardcoded request id?

Reply:
It is an example response of an attempt to access the ORD Service without authorization, all the information is a dummy.